services/gateway/service/auth.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/Drozd0f/ttto-go/gen/proto/auth"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	ErrValidation = errors.New("validation error")
+	ErrUserAlreadyExists   = errors.New("user already exist")
+	ErrUserNotExists     = errors.New("user not exist")
+)
+
+func (s *Service) CreateUser(ctx context.Context, cur *auth.CreateUserRequest) error {
+	_, err := s.ac.CreateUser(ctx, cur)
+	if err != nil {
+		stat, ok := status.FromError(err)
+		if ok {
+			switch {
+			case stat.Code() == codes.InvalidArgument:
+				return ErrValidation
+			case stat.Code() == codes.AlreadyExists:
+				return ErrUserAlreadyExists
+			}
+			return fmt.Errorf("auth service create user: %s", stat.Message())
```

Comment:
Remove it from "OK", error is returning https://github.com/Drozd0f/ttto-go/pull/1/files#diff-588d886a054225b34283565cd06fa903e92c4a0f52783f2235d3310dfc7cc584R33 here